### PR TITLE
ENH Call validate on form fields within form

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10908

### CODE REVIEW

File: src/Forms/Form.php [hunk 1]
```diff
@@ -1264,17 +1266,23 @@ public function getLegend()
      */
     public function validationResult()
     {
+        $result = ValidationResult::create();
         // Automatically pass if there is no validator, or the clicked button is exempt
```
Comment (GuySartorelli) [inline src/Forms/Form.php]:
```suggestion
        // Automatically pass if the clicked button is exempt
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
One very tiny change

File: src/Forms/Form.php (hunk 1 above)
Comment (emteknetnz) [reply src/Forms/Form.php]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**